### ISSUE TITLE
Award Encounter Group not work when need not split XP

### ISSUE THREAD

**Issue by gandb** [foundryvtt/dnd5e#4321]
If you put the 100XP for each member of party, the XP is splited between the characters.

https://github.com/user-attachments/assets/6e8cfec1-8091-43a6-b738-96a00f6754cb

**Comment by arbron** [comment]
Since there is only 100 xp in the group, it can't give them 100 xp each (that would require at least 200 xp in the party), so it just divides up what is left in the group actor.

**Comment by gandb** [comment]
If I need first multiply to after the  tool devide, the tool make no sense

**Comment by LucifurMacomb** [comment]
Sorry, coming back to this after I've also experienced it. 

I agree that the tool does not feel intuitive when based on the options presented. Based purely on the names: **Split** should divide the sum against the total number of selected actors, whereas **Each** should grant the sum to the number of selected actors. This is how it works with Award Enrichers, and so it only reads as an inconsistency regarding the user interfacing with the system. 

If the idea is for the Group actor's Award function to work differently (using a pool,) using the same UI for different outcomes feels like a discrepancy. Just my opinion, of course, but there it is.

**Comment by gandb** [comment]
I agree with you, multiplying the XP and then selecting Split makes no sense at all. Each should do this automatically.
